packages/nextjs/src/utils/instrumentServer.ts

```diff
@@ -1,6 +1,6 @@
 import { deepReadDirSync } from '@sentry/node';
 import { getActiveTransaction, hasTracingEnabled } from '@sentry/tracing';
-import { Event as SentryEvent, Transaction } from '@sentry/types';
```

Comment:
Can we reorganize the file paths so this isn’t under utils anymore? Feel like it’s server specific so we can make a folder for server/instrument.ts

Reply:
I'm not opposed to rethinking the overall organization scheme, but let's do that in a separate PR.